Gordon360/ApiControllers/ScheduleController.cs

```diff
@@ -113,10 +113,10 @@ public IHttpActionResult Get(string username)
                         break;
                     case Position.STUDENT:
                         var stuProfile = _profileService.GetStudentProfileByUsername(username);
-                        if (stuProfile.IsSchedulePrivate == 0)
-                        {
+                        // if (stuProfile.IsSchedulePrivate == 0)
```

Comment:
Should we remove this commented out code?

Reply:
This code is used after we put privacy settings.